.github/workflows/geth-with-protocol.yml

```diff
@@ -0,0 +1,30 @@
+name: geth-with-protocol
+on:
+  push:
+    branches:
+      - "master"
```

Comment:
I think this will rebuild the image after every push to `master`.
- If we want to do it, then the image should be `livepeer/geth-with-livepeer-protocol:latest` (not `confluence`)
- If we don't want to do it, then I'd set the GH Action to trigger manually (you can also consider building on pushing a tag, but the issue here is that this repo contains a lot of other things, so I'm not sure we want to tag only for `livepeer/geth-with-livepeer-protocol`

Reply:
good point, i think that the second option is the most sensible. also checking out the old [geth-with-livepeer](https://hub.docker.com/r/livepeer/geth-with-livepeer-protocol/tags) it doesn't seem it was following any particular tag system other than a new release for a breaking change